src/yum-plugins/enabled_repos_upload.py

```diff
@@ -5,14 +5,16 @@
 
 from logging import Logger
 
-from katello.utils import combined_profiles_enabled
+from katello.utils import combined_profiles_enabled, is_root_user
 from katello.enabled_report import EnabledReport
 from katello.constants import REPOSITORY_PATH
 
 requires_api_version = '2.3'
 plugin_type = (TYPE_CORE, TYPE_INTERACTIVE)
 
 def close_hook(conduit):
+    if not is_root_user():
```

Comment:
I think performing a yum action is now going to show this message twice, right? once for yum, and once for our plugin here. No need to mention it twice IMO, so why not just return from this method without returning anything?

Reply:
@jturel yum action does not throw any message for a non-root user. 
```
[vagrant@centos7-katello-client ~]$ yum search htop
Loaded plugins: enabled_repos_upload, fastestmirror, package_upload, product-id, search-disabled-repos, subscription-manager
Loading mirror speeds from cached hostfile
 * base: mirror.math.princeton.edu
 * epel: epel.mirror.constant.com
 * extras: mirror.dal10.us.leaseweb.net
 * updates: mirror.mi.incx.net
===================================================================================== N/S matched: htop ======================================================================================
htop.x86_64 : Interactive process viewer

  Name and summary matches only, use "search all" for everything.
You need to be root to perform this command.
```
Should the message say: `You need to be root to upload Enabled Repositories Report.`

Reply:
OK! I see why you left a message.

How about "Skipping Enabled Repositories Upload because you are not logged in as root" ? Gives a little more indication that the activity was skipped, and things kept on going normally otherwise